is_left_camera: false

acquisition_frame_rate: &frame_rate 10.0

# Strobe and trigger Settings
trigger_selector: "FrameStart"
trigger_source: "Line3"
trigger_activation_mode: "FallingEdge"
enable_trigger: true


# Diagnostics configuration:
desired_freq: *frame_rate
